<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabLayer">
      <attribute name="title">
       <string>Linked layer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="WDocNodesViewTBar" name="wDocNodesViewTBar" native="true"/>
       </item>
       <item>
        <widget class="WDocNodesView" name="tvDocNodesView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabComment">
      <attribute name="title">
       <string>Description</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="WTextEdit" name="wDescription" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbOk">
        <property name="text">
         <string>Ok</string>
        </property>
        <property name="icon">
         <iconset theme="dialog-ok">
          <normaloff>../../../../../../../../../../../../../../../../../../../../../../.designer/backup</normaloff>../../../../../../../../../../../../../../../../../../../../../../.designer/backup</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset theme="dialog-cancel">
          <normaloff>../../../../../../../../../../../../../../../../../../../../../../.designer/backup</normaloff>../../../../../../../../../../../../../../../../../../../../../../.designer/backup</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WTextEdit</class>
   <extends>QWidget</extends>
   <header>bulinotes.pktk.widgets.wtextedit</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WDocNodesView</class>
   <extends>QTreeView</extends>
   <header>bulinotes.pktk.widgets.wdocnodesview</header>
  </customwidget>
  <customwidget>
   <class>WDocNodesViewTBar</class>
   <extends>QWidget</extends>
   <header>bulinotes.pktk.widgets.wdocnodesview</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="dark_icons.qrc"/>
  <include location="light_icons.qrc"/>
 </resources>
 <connections/>
</ui>
